src/Facebook/InstantArticles/Transformer/Rules/GlobalRule.php

```diff
@@ -136,6 +139,12 @@ public function apply($transformer, $instantArticle, $node)
             $header->withTime(Time::create(Time::PUBLISHED)->withDatetime($timePublished));
         }
 
+        // Treats Header Image
+        $articleHeaderImageURL = $this->getProperty(self::PROPERTY_GLOBAL_HEADER_IMAGE, $node);
+        if ($articleHeaderImageURL) {
+            $header->withCover(Image::create()->withURL($articleHeaderImageURL));
```

Comment:
If this is the only way we are adding the image to the header, then, we are missing the opportunity to add caption content to this image as well.

Probably you will need to add a call to `$transformer->transform($header, $node)` or to find the caption by some other property you set on the configuration of the GlobalRule as well (some sort of `image.caption` property name to be created).

Reply:
Thanks @everton-rosario. I had spoken with @diegoquinteiro about this and he mentioned that image caption was low priority for now. @diegoquinteiro can you please confirm?

If we are considering image caption along with this change I will make sure make the appropriate code changes.

Reply:
@yeah-sir confirmed! Let's merge this one and follow up with an issue.